Diffing_Engine/Modify/RemoveDuplicatesByHash.cs

```diff
@@ -33,16 +33,21 @@
 using BH.Engine.Serialiser;
 using BH.oM.Reflection.Attributes;
 using System.ComponentModel;
+using BH.Engine.Base;
 
 namespace BH.Engine.Diffing
 {
     public static partial class Modify
     {
-        [Description("Removes duplicates from a collection of objects. The comparison is made through their Diffing Hash.")]
+        [Description("Removes duplicates from a collection of objects. The comparison is made using their Hash. If hash is missing, it is computed.")]
```

Comment:
```suggestion
        [Description("Removes duplicates from a collection of objects. The comparison is made using their Hash. If the hash is missing, it will be computed.")]
```

Reply:
Will the hash be stored on the object that is returned? (Apologies if that's explained somewhere else - but if the answer is yes, probably good to tell the user that in the description as well.)

Reply:
The idea is: if it's not stated anywhere and it's not obvious from the method name, then it's not done 😄